src/foam/u2/wizard/PrerequisiteWAO.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.wizard',
+  name: 'PrerequisiteWAO',
+  implements: [ 'foam.u2.wizard.WAO' ],
+  flags: ['web'],
+  extends: 'foam.u2.wizard.ProxyWAO',
+  todo: [
```

Comment:
Kevin informed me that FOAM1 supported an `issues` axiom. We should use that convention here.
```suggestion
  issues: [
```